pom.xml

```diff
@@ -219,6 +240,32 @@
                             </target>
                         </configuration>
                     </execution>
+                    <execution>
+                        <?m2e execute onConfiguration?>
+                        <id>make-data-dir</id>
+                        <phase>initialize</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <target>
+                                <mkdir dir="${project.basedir}/data"/>
+                            </target>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <?m2e execute onConfiguration?>
+                        <id>copy-setupxml</id>
+                        <phase>initialize</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <target>
+                                <copy file="${project.basedir}/development/jetty/setup.xml" tofile="${project.basedir}/content/erddap/setup.xml" overwrite="true"/>
```

Comment:
Instead of copying over the setup.xml that is provided in the content directory, should we use -DerddapContentDirectory to point jetty to the development/jetty directory? I understand we might need to add a datasets.xml for that. If we're going to use jetty for running local tests, we'll probably want a datasets file for that anyway.

If we do that, I think we can get rid of the entire copy setup step here?

Reply:
I haven't tested running ERDDAP with a content directory that wasn't a clone of what's in GitHub. 

Copying the setup.xml over to the existing content directory seemed like the easiest solution, but if only a datasets.xml file is required alongside setup.xml for ERDDAP to run, then it might be simpler to duplicate both in the ERDDAP repo somewhere, rather than download erddapContent.zip.  

Perhaps that antrun action could be eliminated as well if ERDDAP will run with only a directory containing both setup.xml and datasets.xml? Or is the `/content` directory needed for other development purposes?  The existing development Dockerfile would need to be updated for one.   I'll give it a try.

Reply:
Yeah mirroring docker with a development/jetty/config directory sounds good.

Leave the download of erddapContent.zip for now.

Reply:
Added a local copy of datasets.xml to ./development/jetty/config and moved setup.xml to the same directory.  

Confirmed all works well with the following Jetty run command: 

```
MAVEN_OPTS="-Xms2048m -Xmx2048m" mvn jetty:run -DerddapContentDirectory=development/jetty/config
```